javaScripts/index.js

```diff
@@ -0,0 +1,15 @@
+// Get references to the hamburger button and the mobile menu toggle button
+const hamburger = document.querySelector('.hamburger');
+const menu = document.querySelector('.menu');
+
+// Add an event lister to hamburger button and the menu toogle buttons
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  menu.classList.toggle('active');
+});
+
+// toogle the active class to show/hide the menu 
+document.querySelectorAll('.menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  menu.classList.remove('active');
```

Comment:
- [ ] kindly consider making sure that if a user clicks on one of the nav-links the correct part of your website is displayed to meet the below requirement cause currently it's not displaying the right portfolio part
> When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
hint: you can do so in your HTML file by adding the correct id to each section on each nav link li